### PR TITLE
Rvc3

### PR DESCRIPTION
**Pull request by HonzaCuhel**

### CODE REVIEW

File: cli/blobconverter/__init__.py [hunk 1]
```diff
@@ -95,7 +96,7 @@ def build(self):
 
 
 __defaults = {
-    "url": "https://blobconverter.luxonis.com",
+    "url": "https://dev-blobconverter.luxonis.com",
```
Comment (tersekmatija) [inline cli/blobconverter/__init__.py:99]: Things like defaults here should fall to https://blobconverter not the dev one.

Might make sense to create a separate branch "rvc3-merge" and refactor those, then make a PR?

File: model.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+description: empty
```
Comment (tersekmatija) [inline model.yml:1]: Here by mistake, should be removed?

File: models/intel_2022_1_RVC3.json
Comment (tersekmatija) [inline models/intel_2022_1_RVC3.json:1]: Please check which are unused and remove.

**Review by tersekmatija — CHANGES_REQUESTED** [review]

File: cli/blobconverter/__init__.py (hunk 1 above)
Comment (HonzaCuhel) [reply cli/blobconverter/__init__.py:99]: Alright, will do

File: model.yml (hunk 2 above)
Comment (HonzaCuhel) [reply model.yml:1]: You mean the whole file or just this one line?

**Pull request closed without merging**